src/idb_companion.js

```diff
@@ -0,0 +1,42 @@
+var childProcess = require('child_process');
+var process = require('process');
+
+export default {
+    boot (udid, timeout) {
+        var { rc, stdout } = this._exec(['--boot', udid], { timeout });
```

Comment:
Let's swap out "var" references with "const".

Reply:
^ I'll make this optional seeing as in the project already started using var.

Maybe a next PR that adds lint rules + adds corrections to failed lint rules would be a good approach.